.github/workflows/integration.yml

```diff
@@ -69,7 +69,10 @@ jobs:
           mv ../${{steps.extract-tikv.outputs.destination}}/tikv-server .
           ./pd-server > pd.log 2>&1 &
           sleep 5
-          ./tikv-server -C tikv.toml > tikv.log 2>&1 &
+          TIKV_COUNT=3
+          for i in $(seq $TIKV_COUNT); do
+            ./tikv-server -C tikv.toml > "tikv${i}.log" 2>&1 &
```

Comment:
Will this cause the 3 TiKV nodes to use the same configuration? The listen address, data path, etc, may be conflicted.

Reply:
Oh, good catch! changed it for diff binding, PTAL again, thx!